js/index.js

```diff
@@ -50,10 +50,30 @@ for (let i = 0; i < skills.length; i++) {
     document.getElementById("submit").reset();
   }
 
+let githubRequest= new XMLHttpRequest();
+githubRequest.open('GET', "https://api.github.com/users/{StephAlvarado}/repos");
+githubRequest.send();
 
+// did I add the correct link to open?
 
+githubRequest.addEventListener('load', () =>{
+let repositories = JSON.parse(githubRequest.response);
+console.log(repositories);
+});
```

Comment:
The variable repositories is only declared within the scope of the event listener for the load event. If you are using it outside that scope (as in the last loop), it will be undefined. You should move the loop inside the load event listener.
Please move brackets }); to line 75 after loop.